pkg/image/docker/tag.go

```diff
@@ -0,0 +1,47 @@
+/*
+Copyright the Sonobuoy contributors 2019
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package docker
+
+import (
+	"time"
+
+	log "github.com/sirupsen/logrus"
+
+	"github.com/heptio/sonobuoy/pkg/image/exec"
+)
+
+// Tag tags an image, retrying up to retries times
+func Tag(src, dest string, retries int) error {
+	log.Infof("Tagging image: %s as %s ...", src, dest)
+	err := exec.Command("docker", "tag", src, dest).Run()
+	// retry tagging up to retries times if necessary
+	if err != nil {
+		for i := 0; i < retries; i++ {
+			time.Sleep(time.Second * time.Duration(i+1))
+			log.WithError(err).Infof("Trying again to tag image: %s as %s ...", src, dest)
+			// TODO(stevesloka): add some backoff / sleep?
```

Comment:
Outdated; looks like you opted for a linear backoff? Only one retry though so its not that big a deal either way.

Reply:
There's no exponential backoff, it just tries once, then retry again.  A real exponential would try, fail, try again and wait a minute, then fail, wait 5 mins, etc.

Reply:
I know its not an exponential backoff; you have a linear back off 2 lines above:
```
time.Sleep(time.Second * time.Duration(i+1))
```